sfge/src/main/java/com/salesforce/exception/SfgeInterruptedException.java

```diff
@@ -5,4 +5,8 @@
  * stop. Any long running methods should periodically invoke {@link Thread#interrupted()} and throw
  * this exception if appropriate.
  */
-public final class SfgeInterruptedException extends SfgeRuntimeException {}
+public final class SfgeInterruptedException extends SfgeRuntimeException {
+    public SfgeInterruptedException() {
+        super();
```

Comment:
isn't this implied?

Reply:
I didn't realize that, but yes it is. That said, I still think it's worth explicitly doing here, to avoid any weird problems in the future.